storybook/pages/CommunityProfilePopupInviteFriendsPanelPage.qml

```diff
@@ -3,85 +3,53 @@ import QtQuick.Controls 2.14
 
 import AppLayouts.Chat.panels.communities 1.0
 import utils 1.0
+import StubDecorators 1.0
 
 Item {
-    property bool globalUtilsReady: false
-    property bool mainModuleReady: false
-
-    QtObject {
-        function isCompressedPubKey(publicKey) {
-            return true
-        }
-
-        function getCompressedPk(publicKey) { return "zx3sh" + publicKey }
-
-        function getColorHashAsJson(publicKey) {
+    UtilsDecorator {
+        globalUtils.isCompressedPubKey: function(publicKey) { return true }
+        globalUtils.getCompressedPk: function(publicKey) { return "zx3sh" + publicKey }
+        globalUtils.getColorHashAsJson: function(publicKey) {
             return JSON.stringify([{colorId: 0, segmentLength: 1},
                                    {colorId: 19, segmentLength: 2}])
         }
-
-        Component.onCompleted: {
-            Utils.globalUtilsInst = this
-            globalUtilsReady = true
-
-        }
-        Component.onDestruction: {
-            globalUtilsReady = false
-            Utils.globalUtilsInst = {}
-        }
     }
 
-    QtObject {
-        function getContactDetailsAsJson() {
-            return JSON.stringify({})
-        }
-
-        Component.onCompleted: {
-            mainModuleReady = true
-            Utils.mainModuleInst = this
-        }
-        Component.onDestruction: {
-            mainModuleReady = false
-            Utils.mainModuleInst = {}
-        }
-    }
+    SharedRootStoreDecorator {}
 
     Frame {
         anchors.centerIn: parent
 
-        Loader {
```

Comment:
Loader was used here on purpose to make sure that `globalUtilsReady` and `mainModuleReady` is prepared before being used by the component. Otherwise the order is undetermined.

Reply:
Good point. Will update